source/includes/common-mc-admin-config.rst

```diff
@@ -1007,3 +1007,20 @@ Specify the maximum limit for undelivered messages. Defaults to ``100000``.
 Specify a comment to associate with the Webhook configuration.
 
 .. end-minio-notify-webhook-comment
+
+.. Root API Access
+
+.. start-minio-root-api-access
+
+.. versionadded:: MinIO Server RELEASE.2023-05-04T21-44-30Z
+
+Specify ``on`` to enable and ``off`` to disable the :ref:`root <minio-users-root>` user account.
+Disabling the root service account also disables all service accounts associated with root, excluding those used by site replication.
```

Comment:
@harshavardhana are there are any other internal services which would continue to support root and/or its service accounts beyond site replication?